src/common/components/cards/fix-instruction-color-box.scss

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+@import '../../../common/styles/colors.scss';
```

Comment:
we have duplicated this code everywhere. I have created this new file for styling color box. We need to import this file in all other usages. but, using this new file in other places is not part of this pr.